From eb5972ccf61707c9549cea7d2551ed8f904158fb Mon Sep 17 00:00:00 2001
From: Simon Ser <contact@emersion.fr>
Date: Thu, 1 Jul 2021 14:38:53 +0200
Subject: [PATCH] Bump libliftoff to 0.1.0

---
 src/drm.cpp            | 4 +++-
 subprojects/libliftoff | 2 +-
 2 files changed, 4 insertions(+), 2 deletions(-)

diff --git a/src/drm.cpp b/src/drm.cpp
index 52dea66..da64aec 100644
--- a/src/drm.cpp
+++ b/src/drm.cpp
@@ -556,6 +556,8 @@ int init_drm(struct drm_t *drm, const char *device, const char *mode_str, unsign
 	drm->lo_device = liftoff_device_create( drm->fd );
 	drm->lo_output = liftoff_output_create( drm->lo_device, drm->crtc_id );
 
+	liftoff_device_register_all_planes( drm->lo_device );
+
 	assert( drm->lo_device && drm->lo_output );
 
 	for ( int i = 0; i < k_nMaxLayers; i++ )
@@ -966,7 +968,7 @@ drm_prepare_liftoff( struct drm_t *drm, const struct Composite_t *pComposite, co
 	{
 		for ( int i = 0; i < k_nMaxLayers; i++ )
 		{
-			if ( liftoff_layer_get_plane_id( drm->lo_layers[ i ] ) != 0 )
+			if ( liftoff_layer_get_plane( drm->lo_layers[ i ] ) != NULL )
 				scanoutLayerCount++;
 		}
 		ret = scanoutLayerCount == pComposite->nLayerCount;
-- 
2.31.1

