From 573ea6573d7603ea9a348741fa4b1bc93235f4c1 Mon Sep 17 00:00:00 2001
From: Neal Gompa <ngompa13@gmail.com>
Date: Sun, 4 Oct 2020 15:16:31 -0400
Subject: [PATCH 1002/1002] meson: Switch to using external libliftoff

---
 meson.build | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/meson.build b/meson.build
index c73f299..55a876c 100644
--- a/meson.build
+++ b/meson.build
@@ -59,9 +59,7 @@ spirv_shader = custom_target('shader_target',
   install : false,
 )
 
-liftoff_proj = subproject('libliftoff', default_options:
-  ['default_library=static'])
-libftoff_dep = liftoff_proj.get_variable('liftoff')
+libftoff_dep = dependency('liftoff')
 
 executable(
     'gamescope',
-- 
2.26.2

