diff -rup gamescope-3.14.2.orig/src/drm.cpp gamescope-3.14.2/src/drm.cpp
--- gamescope-3.14.2.orig/src/drm.cpp	2024-02-21 14:11:46.000000000 -0800
+++ gamescope-3.14.2/src/drm.cpp	2024-06-29 17:15:53.555626848 -0700
@@ -2457,7 +2457,15 @@ drm_prepare_liftoff( struct drm_t *drm,
 		}
 	}
 
+#ifdef HAVE_LIBLIFTOFF_0_5
+	struct liftoff_output_apply_options lo_options = {
+		.timeout_ns = std::numeric_limits<int64_t>::max()
+	};
+
+	int ret = liftoff_output_apply( drm->lo_output, drm->req, drm->flags, &lo_options );
+#else
 	int ret = liftoff_output_apply( drm->lo_output, drm->req, drm->flags );
+#endif
 
 	if ( ret == 0 )
 	{
diff -rup gamescope-3.14.2.orig/src/meson.build gamescope-3.14.2/src/meson.build
--- gamescope-3.14.2.orig/src/meson.build	2024-06-29 17:01:44.332069304 -0700
+++ gamescope-3.14.2/src/meson.build	2024-06-29 17:07:59.697303990 -0700
@@ -32,7 +32,7 @@ wlroots_dep = dependency(
 
 displayinfo_dep = dependency(
   'libdisplay-info',
-  version: ['>= 0.0.0', '< 0.2.0'],
+  version: ['>= 0.0.0', '< 0.3.0'],
   fallback: ['libdisplay-info', 'di_dep'],
   default_options: ['default_library=static'],
 )
@@ -113,7 +113,7 @@ if drm_dep.found()
   required_wlroots_features += 'libinput_backend'
   liftoff_dep = dependency(
     'libliftoff',
-    version: ['>= 0.4.0', '< 0.5.0'],
+    version: ['>= 0.4.0', '< 0.6.0'],
     fallback: ['libliftoff', 'liftoff'],
     default_options: ['default_library=static'],
   )
@@ -121,6 +121,10 @@ else
   liftoff_dep = dependency('', required: false)
 endif
 
+if liftoff_dep.found() and liftoff_dep.version().version_compare('>=0.5.0')
+	gamescope_cpp_args += '-DHAVE_LIBLIFTOFF_0_5'
+endif
+
 if sdl_dep.found()
   src += 'sdlwindow.cpp'
 endif
